web-console/src/druid-models/lookup-spec/lookup-spec.tsx

```diff
@@ -464,30 +464,30 @@ export const LOOKUP_FIELDS: Field<LookupSpec>[] = [
   {
     name: 'extractionNamespace.jitterSeconds',
     type: 'number',
-    placeholder: '30 (optional), default is 0',
+    placeholder: '30 (optional) ',
```

Comment:
instead of setting the placeholder this you could just add `defaultValue: 30` this basically sets the placeholder to `30`. The `(optional)` label is not needed because visually when you open the form and there is a default value filled in as the placeholder it visually conveys "optional". The placeholder is really there for when you need to override the default behavior that I don't think you need to do here.